pkg/apis/pipeline/v1beta1/taskrun_validation_test.go

```diff
@@ -515,7 +515,31 @@ func TestTaskRunSpec_Invalidate(t *testing.T) {
 		},
 		wantErr: apis.ErrMissingField("sidecarOverrides[0].name"),
 		wc:      enableAlphaAPIFields,
+	}, {
+		name: "invalid both step-level (stepOverrides.resources) and task-level (spec.computeResources) resource requirements",
```

Comment:
could you please add a test case that ensures that you can configure both sidecarOverrides and computeResources?

Reply:
sure! thanks